### ISSUE TITLE
⛔ 🔗 Editor Top Bar feature page exit choices don't work (Duplicate of #22359)

### ISSUE THREAD

**Issue by nickdgreen** [elementor/elementor#25089]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

If the Editor Top Bar feature is Active then the page exit choices This Post, All Posts, and WP Dashboard under user preferences have no effect and the page always exits the the default page editor of that page.

### Steps to reproduce

Activate the Editor Top Bar feature
Open a page in the Elementor Editor
Click on the Exit icon
Click on User Preferences
In "Exit to" Choose WP Dashboard
Publish the change
Click on Exit to WordPress
Page does not exit to the dashboard

### Expected behavior

Activate the Editor Top Bar feature
Open a page in the Elementor Editor
Click on the Exit icon
Click on User Preferences
In "Exit to" Choose WP Dashboard
Publish the change
Click on Exit to WordPress
The page exists to the dashboard

I know the system file I have provided is from a development system but it is the same on an ordinary installation. I am also aware that the Editor Top Bar is in Alpha. This a bug report for the developers.

### Isolating the problem

- [X] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Percona Server (GPL), Release 23, Revision 500fcf5 v5.7.23-23
	PHP Version: 8.2.15
	PHP Memory Limit: 512M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 516M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.4.3
	Site URL: https://elementary.websitesdesigned4u.com
	Home URL: https://elementary.websitesdesigned4u.com
	WP Multisite: No
	Max Upload Size: 512 MB
	Memory limit: 512M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en-GB
	Timezone: Europe/London
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 3.0.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36

== Active Plugins ==
	Akismet Anti-spam: Spam Protection
		Version: 5.3.1
		Author: Automattic - Anti-spam Team

	Code Snippets
		Version: 3.6.2
		Author: Code Snippets Pro

	Code Snippets Pro (Premium)
		Version: 3.6.3
		Author: Code Snippets Pro

	Complianz - Terms and Conditions
		Version: 1.2.7
		Author: Really Simple Plugins

	Complianz | GDPR/CCPA Cookie Consent
		Version: 6.5.6
		Author: Really Simple Plugins

	Elementor
		Version: 3.19.0-dev6
		Author: Elementor.com

	Elementor Beta (Developer Edition)
		Version: 1.1.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.19.2
		Author: Elementor.com

	ManageWP - Worker
		Version: 4.9.19
		Author: GoDaddy

	PhastPress
		Version: 3.0
		Author: Albert Peschar

	Rank Math SEO
		Version: 1.0.212.1
		Author: Rank Math

	Rank Math SEO PRO
		Version: 3.0.54
		Author: Rank Math

	Really Simple SSL
		Version: 7.2.2
		Author: Really Simple Plugins

	Solid Security Basic
		Version: 9.3.1
		Author: SolidWP

	UpdraftPlus - Backup/Restore
		Version: 1.23.16
		Author: UpdraftPlus.Com, DavidAnderson

	WP Meteor
		Version: 3.3.3
		Author: Aleksandr Guidrevitch


== Must-Use Plugins ==
	Endurance Page Cache
		Version: 2.2
		Author: Mike Hansen

	ManageWP - Worker Loader
		Version: 1.0.0
		Author: GoDaddy


== Elements Usage ==
	
	container : 6
		button : 1
		container : 54
		form : 2
		heading : 5
		html : 36
		image : 1
		nested-accordion : 3
		nested-tabs : 1
	header : 1
		container : 1
		image : 1
	page : 2
		heading : 4
		text-editor : 5
	popup : 1
		container : 2
		form : 1
		heading : 2
	widget : 0
		button : 1
		icon : 1
	wp-post : 2
		button : 1
		container : 1
		text-editor : 2
	wp-page : 10
		button : 4
		container : 88
		divider : 1
		form : 3
		heading : 32
		html : 28
		icon-box : 3
		icon-list : 1
		image : 14
		image-box : 6
		menu-anchor : 1
		nav-menu : 2
		nested-accordion : 3
		nested-tabs : 2
		shortcode : 1
		social-icons : 2
		text-editor : 18
	single-post : 2
		button : 1
		container : 6
		heading : 1
		post-info : 1
		post-navigation : 1
		posts : 1
		share-buttons : 1
		theme-post-content : 1
		theme-post-title : 1


== Settings ==
	
	cpt_support: post, page
	allow_tracking: yes
	unfiltered_files_upload: 1
	load_fa4_shim: yes


== Features ==
	Custom Fonts: 1
	Custom Icons: 0

== Integrations ==
	
	recaptcha_v3: Active


== Elementor Experiments ==
	Improved asset loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Additional Custom Breakpoints: Active
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Grid Container: Active
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Editor Top Bar: Active
	Optimized Gutenberg Loading: Active by default
	Build with AI: Active
	Landing pages: Active by default
	Nested Elements: Active
	Lazy Load Background Images: Inactive by default
	Optimize Image Loading: Active by default
	Pages Panel: Inactive by default
	Global Style Guide: Active
	Page Transitions: Active by default
	Notes: Active by default
	Display Conditions: Active
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Active
	Taxonomy Filter: Inactive by default


== Log ==
	
JS: showing 2 of 2JS: 2024-01-19 12:13:01 [error X 1][../wp-includes/js/jquery/jquery.min.js?ver=3.7.1:2:28760] elementor is not defined 
JS: 2024-01-19 13:12:38 [error X 2][../wp-admin/load-scripts.php?c=0&loadchunk_0=jquery-core,jquery-migrate,utils,wp-polyfill-inert,regenerator-runtime,wp-polyfill,wp-hooks&ver=6.4.2:2:28760] Cannot read properties of undefined (reading 'value') 

PHP: showing 3 of 3PHP: 2024-01-20 11:05:41 [notice X 10][../wp-content/plugins/elementor/includes/template-library/sources/local.php::780] ob_end_clean(): Failed to delete buffer. No buffer to delete [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-01-23 16:10:22 [notice X 25][../wp-content/plugins/elementor/includes/base/controls-stack.php::228] hexdec(): Passing null to parameter #1 ($hex_string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-02-06 15:42:49 [warning X 1][../wp-content/plugins/elementor/includes/managers/image.php::113] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 20 of 372024-01-24 21:24:53 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.19.0-dev5',
  'to' => '3.19.0-dev6',
)]
2024-01-25 12:11:36 [info] elementor-pro::elementor_pro_updater Started 
2024-01-25 12:11:36 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-01-25 12:11:36 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-01-25 12:11:36 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.19.0-beta5',
  'to' => '3.19.0-beta6',
)]
2024-01-31 20:07:45 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.19.0-beta6',
  'to' => '3.19.0',
)]
2024-01-31 20:09:11 [info] elementor-pro::elementor_pro_updater Started 
2024-01-31 20:09:11 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-01-31 20:09:11 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-01-31 20:09:11 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.19.0-beta6',
  'to' => '3.19.0',
)]
2024-02-06 19:18:11 [info] elementor-pro::elementor_pro_updater Started 
2024-02-06 19:18:11 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-02-06 19:18:11 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-02-06 19:18:11 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.19.0',
  'to' => '3.19.1',
)]
2024-02-06 19:18:11 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.19.0',
  'to' => '3.19.1',
)]
2024-02-06 19:18:12 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.19.0',
  'to' => '3.19.1',
)]
2024-02-11 16:53:31 [info] elementor-pro::elementor_pro_updater Started 
2024-02-11 16:53:31 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-02-11 16:53:31 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-02-11 16:53:31 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.19.1',
  'to' => '3.19.2',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hi, @nickdgreen,

Thank you for reaching out! 🙏 

### ⛔ After reviewing your report we concluded this is not a bug caused by Elementor. The feature you're referring to is not yet supported.

If you think this should be an Elementor feature, please cast your ⬆️ **Upvote** in #22828 and we will consider it in the future if there is enough demand (votes) from other users.

Kind regards

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.22.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
